.github/workflows/create-release-branch.yml

```
@@ -0,0 +1,76 @@
+name: Create release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      patch_version:
+        description: Patch version (integer)
+        required: true
+env:
+  NODE_ENV: production
+  RELEASE_VERSION: '${{ format(''{0}.{1}'', github.ref_name, github.event.inputs.patch_version) }}'
```

This doesn't look formatted.

---

What do you mean?

---

I mean that it looks like your format-on-save isn't working (at least, for yml files).

```suggestion
  RELEASE_VERSION: "${{ format('{0}.{1}', github.ref_name, github.event.inputs.patch_version) }}"
```

---

Don't forget to fix this before merging.

---

Ideally these kind of things should be addressed _before_ requesting another review, so that we don't waste time requesting the same changes again.